container/rkt/install/install.go

```diff
@@ -26,6 +26,14 @@ import (
 
 func init() {
 	err := container.RegisterPlugin("rkt", container.PluginFn{
```

Comment:
(optional) I have a slight preference for implementing the interface with an empty struct, since it reduces the amount of indentation here, although it does add a bit of boilerplate. Your call.

Reply:
done, implemented in empty structs in the main provider package, accessible via `NewPlugin()`